core/app/modules/site-editor/assets/js/organisms/error-dialog.js

```diff
@@ -8,34 +8,38 @@ import DialogContent from '../ui/dialog/dialog-content';
 export default function ErrorDialog( props ) {
 	return (
 		<Dialog>
-			<>
-				<DialogContent>
-					<>
-						<DialogTitle>{ props.title }</DialogTitle>
-						<DialogText>{ props.text }</DialogText>
-					</>
-				</DialogContent>
-				<DialogActions>
-					<>
-						<DialogButton text="Go Back" onClick={ props.goBackHandler } tabIndex="2"/>
-						<DialogButton text="Learn More" url={ props.learnMoreUrl } tabIndex="1" color="link" target="_blank"/>
-					</>
-				</DialogActions>
-			</>
+			<DialogContent>
+				<DialogTitle>{ props.title }</DialogTitle>
+				<DialogText>{ props.text }</DialogText>
+			</DialogContent>
+			<DialogActions>
+				<DialogButton text={ props.backButtonText } onClick={ props.backButtonHandler } tabIndex="2"/>
+				<DialogButton
+					text={ props.learnButtonText }
+					url={ props.learnButtonUrl }
+					tabIndex="1"
+					color="link"
+					target="_blank"
+				/>
+			</DialogActions>
 		</Dialog>
 	);
 }
 
 ErrorDialog.propTypes = {
 	title: PropTypes.string,
 	text: PropTypes.string,
-	learnMoreUrl: PropTypes.string,
-	goBackHandler: PropTypes.func,
+	learnButtonUrl: PropTypes.string,
+	backButtonHandler: PropTypes.func,
+	learnButtonText: PropTypes.string,
+	backButtonText: PropTypes.string,
```

Comment:
Reorder here and on defaultProps.

```suggestion
	learnButtonText: PropTypes.string,
	learnButtonUrl: PropTypes.string,
	backButtonText: PropTypes.string,
	backButtonHandler: PropTypes.func,
```